fonts/Makefile.am

```diff
@@ -120,6 +120,10 @@ dist_font_DATA = \
 	Skranji-Regular.ttf \
 	Sniglet-ExtraBold.ttf \
 	Sniglet-Regular.ttf \
+	SutonnyMJ.ttf \
+	SutonnyMJ-Bold.ttf \
+	SutonnyMJ-BoldItalic.ttf \
+	SutonnyMJ-Italic.ttf \
```

Comment:
We just had a call with the mPower folks, and they say that the font is freely available for anyone to use, and that there's no license associated with it. Consulted with @ptomato and he thinks it's ok to add it.